### ISSUE TITLE
[Bug] Settings category are not read by Voice Assistant on Samsung devices

### ISSUE THREAD

**Issue by mcarare** [mozilla-mobile/fenix#7307]
### Device information

* Android device:  Samsung devices with Voice Assistant instead of Talkback
* Fenix version: 3.0.0 and latest master

**Comment by sblatz** [comment]
I cannot reproduce this on my Galaxy S9 Android 9. QA can we please verify this is still happening?

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **`Nightly Build #20290608 from 1/29`** using the following devices:
• Samsung Galaxy S8+(Android 9)
• Samsung Galaxy S7 (Android 7)
